iron-form.html

```diff
@@ -271,14 +295,14 @@
         // elements with those names.
         this.ajax.url = this._form.getAttribute('action');
         this.ajax.method = this._form.getAttribute('method');
-        this.ajax.contentType = this._form.contentType;
-        //this.ajax.withCredentials = this.withCredentials;
-        //this.ajax.headers = this.headers;
+        this.ajax.contentType = this._form.getAttribute('enctype');
+        this.ajax.withCredentials = this.withCredentials;
+        this.ajax.headers = this.headers;
 
-        if (this._form.method.toUpperCase() === 'POST') {
-          this.ajax.body = json;
-        } else {
+        if (this.ajax.method.toUpperCase() === 'GET') {
           this.ajax.params = json;
```

Comment:
Oh it does, I somehow thought it doesn't without actually confirming. This document says it's not in active maintenance though (not sure what it means exactly). Anyway, let me know what you decide about PUT/DELETE.